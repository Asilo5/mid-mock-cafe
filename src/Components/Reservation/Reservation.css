.reservation {
    background: white;
    padding: 20px;
    margin: 20px;
}