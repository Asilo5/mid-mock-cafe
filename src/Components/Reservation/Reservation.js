import React from 'react';

const Reservation = ({id, name, date, time, number}) => {
  return (
      <section>
        <h2>{name}</h2>
        <p>{date}</p> 
        <p>{time}</p>
        <p>Number of guests: {number} </p>
        <button>Cancel</button>
      </section>
  )
}

export default Reservation;