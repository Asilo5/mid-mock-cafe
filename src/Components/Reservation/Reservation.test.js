import React from 'react';
import { shallow } from 'enzyme';
import Reservation from './Reservation';

describe('Reservation', () => {
    it('should match wrapper with all information passing through correctly', () => {
        let wrapper = shallow(<Reservation id={1} 
                                          name='bob' 
                                          date='11/22' 
                                          time='7:30'
                                          number={5}
                                          removeReservations={jest.fn()} />);
     expect(wrapper).toMatchSnapshot();   
    })
})