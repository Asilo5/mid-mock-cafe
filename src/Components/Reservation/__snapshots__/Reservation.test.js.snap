// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation should match wrapper with all information passing through correctly 1`] = `
<section
  className="reservation"
>
  <h2>
    bob
  </h2>
  <p>
    11/22
  </p>
  <p>
    7:30
  </p>
  <p>
    Number of guests: 
    5
     
  </p>
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</section>
`;
